generators/_base/BaseSubGenerator.js

```diff
@@ -105,12 +128,94 @@ module.exports = class BaseSubGenerator extends BaseGenerator {
     );
   }
 
+  /**
+   * Transform name.
+   *
+   * @param {string} name - Name to transform.
+   * @returns {string} - Transformed name.
+   */
+  transformName(name) {
+    // todo maybe this method should be private
```

Comment:
This method is default transform. Developer may override it inside own generator.